aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -319,14 +319,32 @@ void slow_conv2d_backward_out_cpu_template(
   });
 }
 
+template <typename scalar_t>
 void slow_conv2d_backward_weight_frame(
-    Tensor& grad_weight,
-    Tensor& grad_output,
-    const Tensor& finput) {
-  auto grad_output_2d = grad_output.view(
-      {grad_output.size(0), grad_output.size(1) * grad_output.size(2)});
-  const Tensor tfinput = finput.transpose(0, 1);
-  grad_weight.addmm_(grad_output_2d, tfinput);
+    TensorAccessor<scalar_t, 2> grad_weight,
+    TensorAccessor<scalar_t, 3> grad_output,
+    TensorAccessor<scalar_t, 2> finput) {
+  // Compute grad_weight += grad_output.reshape({grad_output.shape(0), -1}) * finput.T
+  // Note gemm expects fortran order, so all 3 matrices are transposed.
+  // Swapping argument order cancels this, since C == AB <=> T(C) == T(B)T(A)
+  const int64_t m = finput.size(0);
+  const int64_t n = grad_output.size(0);
+  const int64_t k = grad_output.size(1) * grad_output.size(2);
+
+  constexpr int64_t one = 1;
```

Comment:
this is unused var (this is a merge-blocking error internally)

Reply:
Ah, I added the fixup removing this line to the wrong commit in the stack. Fixed now.